os_stub/platform_lib/time_linux.c

```diff
@@ -26,4 +29,7 @@ void libspdm_sleep(uint64_t milliseconds)
     do {
         err=select(0, NULL, NULL, NULL, &tv);
     } while(err<0 && errno==EINTR);
+#else
+
```

Comment:
Please add comment such as porting needed.
and ASSERT() here.

Reply:
Thanks. I have added comment and ASSERT().